### PR TITLE
Change codecov.yml and set it with default values

### PR DESCRIPTION
**Pull request by ShreyamsJain**

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -1,75 +1,19 @@
...
-  bot: ShreyamsJain
-  token: e0eeb7cb-dba3-40ac-a5b8-770a6c78e485
-
-  ci:
-    -"https://codeship.com/"
-
+#
+# This codecov.yml is the default configuration for
+# all repositories on Codecov. You may adjust the settings
+# below in your own codecov.yml in your repository.
```
Comment (lasley) [inline codecov.yml:4]: You can remove this comment

File: codecov.yml [hunk 2]
```diff
@@ -1,75 +1,19 @@
...
-        only_pulls: null
-
-    irc:
-      default:
-        server: null
-        channel: null
-        branches: null
-        threshold: null
-        message: null
+  range: 70...100
```
Comment (lasley) [inline codecov.yml]: IMO 80...100 is better, but that's up to you guys.

70% is a C, which is average. Do you strive for average or excellence?

File: codecov.yml [hunk 3]
```diff
@@ -1,75 +1,19 @@
...
+    patch: true
+    changes: false
 
 comment:
-  layout: "header, diff, changes, sunburst, uncovered, tree"
-  branches: null
-  behavior: default
\ No newline at end of file
+  layout: "header, diff"
+  behavior: default  # update if exists else create new
```
Comment (lasley) [inline codecov.yml]: All lines should end with a carriage return https://robots.thoughtbot.com/no-newline-at-end-of-file

**Review by lasley — CHANGES_REQUESTED** [review]

File: codecov.yml (hunk 3 above)
Comment (ShreyamsJain) [reply codecov.yml]: @lasley I have performed the necessary changes

**Review by lasley — APPROVED** [review]

**Pull request merged**